docs/content/how-to/hibernate-workers.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Hibernating Hosted Control Plane Worker nodes 
```

Comment:
Maybe we call it scaling cluster to zero nodes. Hibernation could mean more than just scaling and has more nuance (e.g., shutdown vs remove nodes).

Reply:
Yes, let's don't call use the term hibernation in this doc.
This is just a SOP that proves how to temporarily shrink to zero your data plane.

Reply:
Thanks for the feedback :), updating it